src/ui/components/services/storage.ts

```diff
@@ -0,0 +1,21 @@
+import Todo from '../models/todo';
```

Comment:
Ahh, sorry about this.  We cannot nest services under components. I should have said `src/services/storage.ts`.